### PR TITLE
PS-4729 : 8.0 eliminates TRUE and FALSE for bool values, only true an…

### PR DESCRIPTION
**Pull request by george-lorch**
…d false are valid.

- s/TRUE/true   s/FALSE/false

### CODE REVIEW

**Comment by george-lorch** [comment]
https://jenkins.percona.com/view/PS%205.6/job/percona-server-5.6-param/2201/

File: storage/tokudb/hatoku_hton.cc [hunk 1]
```diff
@@ -575,10 +575,10 @@ static int tokudb_init_func(void *p) {
 
     db_env->set_update(db_env, tokudb_update_fun);
 
-    db_env_set_direct_io(tokudb::sysvars::directio == TRUE);
+    db_env_set_direct_io(tokudb::sysvars::directio == true);
```
Comment (laurynas-biveinis) [inline storage/tokudb/hatoku_hton.cc]: cond == true -> cond, but see the comment below

File: storage/tokudb/hatoku_hton.cc [hunk 2]
```diff
@@ -575,10 +575,10 @@ static int tokudb_init_func(void *p) {
...
 
     db_env_set_compress_buffers_before_eviction(
-        tokudb::sysvars::compress_buffers_before_eviction == TRUE);
+        tokudb::sysvars::compress_buffers_before_eviction == true);
```
Comment (laurynas-biveinis) [inline storage/tokudb/hatoku_hton.cc]: cond == true -> cond, but see the comment below

File: storage/tokudb/tokudb_sysvars.cc [hunk 3]
```diff
@@ -48,41 +48,41 @@ const size_t error_buffer_max_size = 1024;
 ulonglong   cache_size = 0;
 uint        cachetable_pool_threads = 0;
 int         cardinality_scale_percent = 0;
-my_bool     checkpoint_on_flush_logs = FALSE;
+my_bool     checkpoint_on_flush_logs = false;
```
Comment (laurynas-biveinis) [inline storage/tokudb/tokudb_sysvars.cc]: false/true are not values of my_bool type, which is char in 5.6. IMHO FALSE/TRUE should be kept for my_bool

**Review by laurynas-biveinis — CHANGES_REQUESTED** [review]

**Comment by george-lorch** [comment]
Made changes to revert the changes to my_bool types, but fixed/kept minor conditional changes. New jenkins https://jenkins.percona.com/view/PS%205.6/job/percona-server-5.6-param/2205/.

Since changes here are trivial and jenkins indicates no issue, merging ahead of further approval.

**Pull request merged**